test/parallel/test-signal-args.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+process.once('SIGINT', common.mustCall((signal) => {
+  assert.strictEqual(signal, 'SIGINT');
+}));
+
+process.kill(process.pid, 'SIGINT');
+
+process.once('SIGTERM', common.mustCall((signal) => {
+  assert.strictEqual(signal, 'SIGTERM');
+}));
+
+process.kill(process.pid, 'SIGTERM');
+
+// Prevent Node.js from exiting due to empty event loop before signal handlers
+// are fired
+setTimeout(() => {}, 0);
```

Comment:
`setImmediate` would be better. Note also that `0` coerces to `1` in timers durations.